### PR TITLE
Update pyVmomi stubs

### PR DESCRIPTION
**Pull request by HansAarneLiblik**
Add some missing pyVmomi stubs. Not all, but some I'm missing

### CODE REVIEW

File: stubs/pyvmomi/pyVmomi/vmodl/__init__.pyi [hunk 1]
```diff
@@ -1,6 +1,25 @@
 from typing import Any
 
+from fault import *
+from query import *
```
Comment (HansAarneLiblik) [inline stubs/pyvmomi/pyVmomi/vmodl/__init__.pyi:4]: Is this the correct way to re-export them? Currently I'm getting
image

Comment (HansAarneLiblik) [reply stubs/pyvmomi/pyVmomi/vmodl/__init__.pyi]: Actually i do not want to re-export them, so this import was unnecessary, but my IDE (IntelliJ 2021.3.3 with Python plugin 213.7172.26) still complains

Comment (srittau) [reply stubs/pyvmomi/pyVmomi/vmodl/__init__.pyi]: How do you import `vmodl`? You could try `from . import query as query` in the stubs.

Comment (HansAarneLiblik) [reply stubs/pyvmomi/pyVmomi/vmodl/__init__.pyi]: In my code: `from pyVmomi import vmodl`

**Comment by sodul** [comment]
@HansAarneLiblik @srittau 

With this changes mypy is now raising:
`error: Exception type must be derived from BaseException`

I suspect that the `class ManagedObjectNotFound:` should be updated to `class ManagedObjectNotFound(Exception):`

Ideally we should have the entire class inheritance:
```
>>> import inspect
>>> inspect.getmro(vmodl.fault.ManagedObjectNotFound)
(<class 'pyVmomi.VmomiSupport.vmodl.fault.ManagedObjectNotFound'>, <class 'pyVmomi.VmomiSupport.vmodl.RuntimeFault'>, <class 'pyVmomi.VmomiSupport.vmodl.MethodFault'>, <class 'pyVmomi.VmomiSupport.vmodl.DynamicData'>, <class 'pyVmomi.VmomiSupport.DataObject'>, <class 'Exception'>, <class 'BaseException'>, <class 'object'>)
```

But fixing the regression would be good. Rolling back with `pip install types-pyvmomi==7.0.5` fixed the new errors. I get the issue with 7.0.6 and 7.0.7.

**Comment by HansAarneLiblik** [comment]
@sodul Yup, `ManagedObjectNotFound` should also derive from `Exception`, my bad!

**Pull request merged**